core/frontend/src/components/vehiclesetup/PwmSetup.vue

```diff
@@ -350,11 +372,17 @@ export default Vue.extend({
     },
     printParam,
     zero_motors() {
+      if (!this.has_focus) {
+        return
```

Comment:
This is a case where we should be safe to run if out of focus no ?

Reply:
I'd prefer to stop completely in case the user is trying to do something else.
We need to check if rover has a decent motor test timeout, actually.

Reply:
But this is zero_motors.. isn't this supposed to be the safest way than letting a undefined state ?

Reply:
we don't want to interfere if the user is using another software. we should probably send one zero_motors and then stop completely

Reply:
Can we have a comment explaining that ? This is not clear from the code.

Reply:
> Can we have a comment explaining that ? This is not clear from the code.

the code wasn't doing it before. I added a boolean to keep track of it.